server/neptune/gateway/event/root_config_builder.go

```diff
@@ -45,9 +47,24 @@ type RootConfigBuilder struct {
 	FileFetcher     fileFetcher
 	GlobalCfg       valid.GlobalCfg
 	Logger          logging.Logger
+	Scope           tally.Scope
 }
 
 func (b *RootConfigBuilder) Build(ctx context.Context, repo models.Repo, branch string, sha string, installationToken int64) ([]*valid.MergedProjectCfg, error) {
+	mergedRootCfgs, err := b.build(ctx, repo, branch, sha, installationToken)
+	if err != nil {
+		b.Scope.Counter(metrics.FilterErrorMetric).Inc(1)
+		return nil, err
+	}
+	if len(mergedRootCfgs) == 0 {
+		b.Scope.Counter(metrics.FilterFailMetric).Inc(1)
```

Comment:
what does fail mean in this context? That implies something bad.  Would it make more sense to just use a gauge here named like `filtered` which would be like a ratio right?

Reply:
Changed it to present/absent after discussion